FastSurferCNN/utils/checkpoint.py

```diff
@@ -142,6 +142,7 @@ def download_checkpoint(download_url, checkpoint_name, checkpoint_path):
     except requests.exceptions.HTTPError as e:
         print('[checkpoint] Response code: {}'.format(e.response.status_code))
```

Comment:
Should this not be logged so it is also in the log files?

Reply:
Yes. We can add a LOGGER instance in `checkpoint.py` to log this message as well.